vnext/Microsoft.ReactNative/Views/TouchEventHandler.cpp

```diff
@@ -194,6 +194,8 @@ void TouchEventHandler::OnPointerConcluded(TouchEventType eventType, const winrt
     m_touchId = 0;
 
   m_xamlView.as<xaml::FrameworkElement>().ReleasePointerCapture(args.Pointer());
+
+  UpdatePointersInViews(args, tag, sourceElement);
```

Comment:
There are probably some edge cases we want to check for here. For example, we do fire UpdatePointersInViews after capturing the pointer in OnPointerPressed. Perhaps we should check that we had previously captured the pointer before calling UpdatePointersInViews?

Reply:
For example, this seems to work:
```
  const auto frameworkElement = m_xamlView.as<xaml::FrameworkElement>();
  const auto wasCaptured =
      frameworkElement.PointerCaptures() != nullptr && frameworkElement.PointerCaptures().Size() > 0;

  m_xamlView.as<xaml::FrameworkElement>().ReleasePointerCapture(args.Pointer());

  // Updates the enter/leave state when pointer was previously captured
  if (wasCaptured) {
    UpdatePointersInViews(args, tag, sourceElement);
  }
```